### PR TITLE
Add deployment for the validation

### PR DESCRIPTION
**Pull request by lbarcziova**

### CODE REVIEW

File: haxxx/validation/job-run-validation.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: run-validation
+spec:
+  schedule: "0 4 * * *"
```
Comment (jpopelka) [inline haxxx/validation/job-run-validation.yaml:6]: Just add a comment describing what this actually means (4AM every day).

File: haxxx/validation/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM fedora:31
+
+RUN dnf install -y python-ogr python-copr python-copr-doc pip && dnf clean all
```
Comment (jpopelka) [inline haxxx/validation/Dockerfile:3]: Do we really need `python-copr-doc`?

```suggestion
RUN dnf install -y python3-ogr python3-copr python3-pip && dnf clean all
```

File: haxxx/validation/openshift/job-run-validation.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                      key: token
+                - name: SENTRY_SECRET
+                  valueFrom:
+                    secretKeyRef:
+                      name: packit-sentry
+                      key: sentry
+              command: ["python3", "/usr/bin/packit-service-validation.py"]
+              volumeMounts:
+                - name: config
+                  mountPath: ~/.config/
```
Comment (jpopelka) [inline haxxx/validation/openshift/job-run-validation.yaml:52]: I think you're missing defining the volume:

```yaml
jobTemplate:
  spec:
    template:
      spec:
        volumes:
          - name: config
            secret:
              secretName: config
```
I'd also suggest renaming `config` to `copr-config`.

File: haxxx/validation/deploy-validation.yaml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+  tasks:
+    - name: Deploy secrets
+      # https://docs.ansible.com/k8s_module.html
+      k8s:
+        namespace: "{{ project }}"
+        definition: "{{ item }}"
+        host: "{{ host }}"
+        api_key: "{{ api_key }}"
+        verify_ssl: "{{ validate_certs }}"
```
Comment (jpopelka) [inline haxxx/validation/deploy-validation.yaml:43]: See https://github.com/packit-service/deployment/pull/83

```suggestion
        validate_certs: "{{ validate_certs }}"
```

**Comment by lbarcziova** [comment]
@jpopelka thanks for the review and suggestions!

**Pull request merged**